types/mongodb/index.d.ts

```diff
@@ -1162,15 +1491,20 @@ export class Mongos extends EventEmitter {
 }
 
 /**
- * @deprecated
```

Comment:
Removed the deprecation warning as it was nowhere to be found on the official docs